### DISCUSSION TITLE
Write tutorials in COLAB

### DISCUSSION THREAD

**Discussion by fernandezc** [spectrochempy/spectrochempy#437] [Developer discussions]
Now that the installation in colab is made easier, I think it would be nice to group the tutorial notebooks in a dedicated repository. I started with a first page with the use of the new features of version 0.4.3.

https://github.com/spectrochempy/spectrochempy_tutorials/blob/master/spectrochempy_tutorial.ipynb

**Comment by atravert** [comment]
It looks good too. Currently the notebooks tutorials are generated by py files  in docs /examples , /userguide and /getting_stated and included in the documentation, which makes 2 types of notebooks: "from doc" and "native colab" . 

- should the "from doc" notebooks be migrated in spectrochempy_tutorials too ?
- should the notebooks in spectrochempy_tutorials appear on the spectrochempy documention ?

**Reply by fernandezc** [reply]
Perhaps we can try to include the Colab notebook in the documentation (we will have to reorganize its construction a bit).

I think it is possible to hide the cells which are specific for colab.

**Reply by fernandezc** [reply]
And so yes some migration is possible, leaving in the user guide only some part. 
We need also to increase the number of short example for the Gallery (this example are visible in the reference section also to illustrate the methods)

**Reply by fernandezc** [reply]
The simplest for now could be  to link part of the documentation to the spectrochempy_tutorials notebooks (and work on a better integration later!)

**Comment by fernandezc** [comment]
For information.

I have updated the COLAB first notebook, with the recommended way to load spectrochempy in development mode (from now it will be from the `develop` branch - not `master`):

```python
!python -m pip install -U git+https://github.com/spectrochempy/spectrochempy.git@develop &>> output.log
exit(0)
```
(note the 0 in the exit function, so that the messages close automatically at the reinitialisation of the runtime environment)
